community/modules/compute/schedmd-slurm-gcp-v6-nodeset/metadata.yaml

```diff
@@ -16,3 +16,5 @@
 spec:
   requirements:
     services: []
+ghpc:
+  inject_module_id: toolkit_module_id
```

Comment:
It seems like this should be set to `name`. Why do the metadata mapping just to create a new variable anyway? If it is because of restrictions in naming then we are going to see that on pretty much every module to some lesser extent. I think we just have to deal with it either through an error or through implicit manipulation of the string.